spec/e2e/default_install_spec.rb

```diff
@@ -18,15 +18,23 @@ def browser
   end
 
   describe 'default webpage' do
-    it 'redirects from root' do
-      browser.goto 'http://localhost:8081/'
+    def expect_page(page)
+      browser.goto "http://localhost:8081#{page}"
```

Comment:
Does this need to be port 80? Here and a few lines below.  I think port 8081 is only going to be used if custom ood_portal.yml is used that changes port number.

Reply:
Didn't you suggest the change? 8081 is the new external port of the container that forwards to 80.

Reply:
Ah right, I keep getting the host vs container port mixed up when looking at the YAML.

Reply:
Actually you'd read the YAML right, I'd misconfigured it.

Reply:
I see - redirects don't work well when container ports don't match.

I'm going to re-implement these tests with curl and assertions on files instead of having to deal with opening the 80 port on the host system.

Reply:
Why not just make the "default" ood_portal.yaml specify a high numbered port we can have match both inside and outside.  We pretty thoroughly test the behavior of empty ood_portal.yml using rspec tests which act a lot like full system tests since they are generating files and that's all the code does.  That seems like path of least resistance. Sometimes acceptance / system tests have to modify some behavior to work within the bounds of systems where tests run.  Will still be able to validate the needs auth page even using like port 9090 or something.

Reply:
This doesn't seem to work. How can I execute commands on the host?
```
on(host, 'curl -o /tmp/page localhost/')
```
I get this error
```
  1) Default install default webpage is the only page
     Failure/Error: on(host, 'curl -o /tmp/page localhost/')
     NoMethodError:
       private method `exec' called for Host "":Serverspec::Type::Host
```

Reply:
Yep and to match other places do it like this:

```
on hosts, "<command here>"
```

If you want to capture the output:

```
on hosts, "<command>" do
  expect(stdout).to match(....)
end
```

Reply:
OK, now tests are failing because dex was expected to be configured everywhere, so most of these have to enable it.

I think the simplest form to enable this automagic install is a simple true.  I'm guessing that'll give us the static ood users for tests.
```yaml
dex: true
```